src/openrct2/actions/LandSetHeightAction.cpp

```diff
@@ -55,6 +55,14 @@ GameActions::Result LandSetHeightAction::Query() const
     StringId errorTitle = CheckParameters();
     if (errorTitle != STR_NONE)
     {
+        if (errorTitle == STR_TOO_LOW)
+        {
+            return GameActions::Result(GameActions::Status::TooLow, errorTitle, STR_NONE);
+        }
+        if (errorTitle == STR_TOO_HIGH)
+        {
+            return GameActions::Result(GameActions::Status::TooHigh, errorTitle, STR_NONE);
+        }
```

Comment:
Have a look at what `errorTitle` holds, and how `LandSetHeightAction::CheckParameters` works.

Not directly related to this PR: both of these identifiers could be renamed to something that makes more sense - `ValidateParameters` for instance. 'Check' is a useless term in programming.